### PR TITLE
Adding `Orange3-Textable`

### PR DESCRIPTION
**Pull request by HeavyTony2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/Orange3-Textable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  imports:
+    - _textable
+    - _textable.widgets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://textable.io
```
Comment (wolfv) [inline recipes/Orange3-Textable/meta.yaml:43]: Is this officially made by textable or thirdparty?

Comment (HeavyTony2) [reply recipes/Orange3-Textable/meta.yaml:43]: This package is basically Textable's "product".

File: recipes/Orange3-Textable/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  summary: Textable add-on for Orange 3 data mining software package.
+  license: GPL-3.0
```
Comment (BastianZim) [inline recipes/Orange3-Textable/meta.yaml:45]: It seems like the license is `GPL-3.0-or-later`: https://github.com/axanthos/orange3-textable/blob/44ffdd36640ba5e92f8a026794bc4583d31cf3a6/setup.py#L11 Do you know which one it is or can you ask upstream about it?

Comment (HeavyTony2) [reply recipes/Orange3-Textable/meta.yaml]: I wrote to the repo owner who told me to use the or-later

File: recipes/Orange3-Textable/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >3
```
Comment (BastianZim) [inline recipes/Orange3-Textable/meta.yaml:21]:
```suggestion
    - python >=3
```

File: recipes/Orange3-Textable/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  run:
+    - appdirs
+    - chardet
+    - future
+    - lttl >=2.0.12
+    - orange3 >=3.14.0
+    - pycountry
+    - python >3
```
Comment (BastianZim) [inline recipes/Orange3-Textable/meta.yaml:29]:
```suggestion
    - python >=3
```

**Review by BastianZim — CHANGES_REQUESTED** [review]

**Comment by BastianZim** [comment]
The rest looks good, just one python thing.

**Comment by BastianZim** [comment]
Those are currently in staged-recipes as well but have no impact on your build – you can just ignore that.

Please add the other python boundary as well and then I can give it a final review.

**Pull request merged**